.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "firmware/spade/src/rpi/no-OS-FatFS"]
```

Comment:
Pretty strongly against adding a submodule into the repo, as it adds complexity that often gets people stuck.  It also leads to the risk of issues added to our project as a result of changes in theirs.  I looked at the license of the project and it doesn't seem problematic to me to just plop the code right into our repo.